dashboard/src/components/partials/CheckDetailsContainer/CheckDetailsConfiguration.js

```diff
@@ -264,9 +280,9 @@ class CheckDetailsConfiguration extends React.PureComponent {
                   </DictionaryValue>
                 </DictionaryEntry>
 
-                <DictionaryEntry>
+                <DictionaryEntry fullWidth={check.envVars.length > 0}>
                   <DictionaryKey>ENV Vars</DictionaryKey>
-                  <DictionaryValue scrollableContent>
```

Comment:
I see that we've sort of replaced scrollableContent with fullWidth, but isn't having the values scroll still desirable in this context?

Reply:
as per IRL discussion; would require a larger refactor which doesn't make sense at this juncture.